docs/reST/ref/draw.rst

```diff
@@ -153,13 +153,57 @@ object around the draw calls (see :func:`pygame.Surface.lock` and
 
 .. function:: arc
 
-   | :sl:`draw a partial section of an ellipse`
-   | :sg:`arc(Surface, color, Rect, start_angle, stop_angle, width=1) -> Rect`
+   | :sl:`draw an elliptical arc`
+   | :sg:`arc(surface=Surface, color=Color, rect=Rect, start_angle=radians, stop_angle=radians) -> Rect`
```

Comment:
Shouldn't `=` be used for default values?

Reply:
I've been using it to indicate named parameters as well. Should it be like this instead?
`arc(surface, color, rect, start_angle, stop_angle) -> Rect`
`arc(surface, color, rect, start_angle, stop_angle, width=1) -> Rect`

Reply:
IMO, that's better. Though I wonder if it would be useful to include some of the RST type info in the docstring somehow?

Reply:
I've made the requested changes. I also opened issue #1020 to track this for the other shape functions.

Regarding the type info, currently the `:param:` and `:type:` directives are used to indicate any type information. Also there is issue #839 which is working on a general type strategy for pygame.